test/unit/org/apache/cassandra/utils/CassandraGenerators.java

```diff
@@ -99,7 +100,7 @@
 
     private static final Gen<SinglePartitionReadCommand> SINGLE_PARTITION_READ_COMMAND_GEN = gen(rnd -> {
         TableMetadata metadata = TABLE_METADATA_GEN.generate(rnd);
-        int nowInSec = (int) rnd.next(Constraint.between(1, Integer.MAX_VALUE));
+        long nowInSec = rnd.next(Constraint.between(1, Cell.MAX_DELETION_TIME - 1));
```

Comment:
Why minus one?

Reply:
Probably a left over from some previous sentinel value collision. Removed and confirmed with CI all is good.